config/php/docker-entrypoint.sh

```diff
@@ -162,6 +162,45 @@ if [ "$1" = 'php-fpm' ] ; then
     >&2 echo "app initialization finished"
 fi
 
+if [ "$1" = 'crond' ] ; then
+    mkdir -p var/cache var/log public/multimedia public/thumbnail public/avatar import export
+    >&2 echo "Setting file permissions..."
+    setfacl -R -m u:www-data:rwX -m u:"$(whoami)":rwX var
```

Comment:
Could we start avoiding ACL's usage as it's not supported on osxfs?

Reply:
ok i can add some if for this